appformer-kogito-bridge/src/main/java/org/appformer/kogito/bridge/client/pmmleditor/marshaller/model/JSIUtils.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.appformer.kogito.bridge.client.pmmleditor.marshaller.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import jsinterop.base.Js;
+import jsinterop.base.JsArrayLike;
+
+public class JSIUtils {
+
+    private JSIUtils() {
+        // Utils class
+    }
+
+    /**
+     * Helper method to create a new, empty <code>JsArrayLike</code>
+     * @return
+     */
+    public static native <D> JsArrayLike<D> getNativeArray() /*-{
+        return [];
+    }-*/;
```

Comment:
This is already in `org.kie.workbench.common.dmn.webapp.kogito.marshaller.mapper.JsUtils`